server.go

```diff
@@ -189,24 +189,30 @@ func (s *StubServer) initializeRouter() error {
 			fmt.Printf("Compiled path: %v\n", pathPattern.String())
 		}
 
-		for verb, method := range verbs {
+		for verb, operation := range verbs {
 			numEndpoints++
 
-			validator, err := getValidator(method)
-			if err != nil {
-				return err
+			requestBodySchema := getRequestBodySchema(operation)
+			var requestBodyValidator *jsval.JSVal
+			if requestBodySchema != nil {
+				var err error
+				requestBodyValidator, err =
+					spec.GetValidatorForOpenAPI3Schema(requestBodySchema)
+				if err != nil {
+					return err
+				}
```

Comment:
This is a little subjective, but this code is starting to get indented pretty deeply. Could we maybe introduce a helper function or something?